src/main/java/frc/robot/commands/Arm/ArmDownCommand.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands.Arm;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ArmSubsystem;
+
+public class ArmDownCommand extends CommandBase {
```

Comment:
worth noting here that we should be using `CommandBase` because the `Command` interface is designed for if you need extra implementation flexibility